bentoml/utils/log.py

```diff
@@ -101,16 +126,22 @@ def get_logging_config_dict(logging_level, base_log_directory):
 
 
 def configure_logging(logging_level=None):
-    if logging_level is None:
-        logging_level = config("logging").get("LEVEL").upper()
-        if "LOGGING_LEVEL" in config("logging"):
-            # Support legacy config name e.g. BENTOML__LOGGING__LOGGING_LEVEL=debug
-            logging_level = config("logging").get("LOGGING_LEVEL").upper()
-
-    if get_debug_mode():
-        logging_level = logging.getLevelName(logging.DEBUG)
-
     base_log_dir = os.path.expanduser(config("logging").get("BASE_LOG_DIR"))
     Path(base_log_dir).mkdir(parents=True, exist_ok=True)
-    logging_config = get_logging_config_dict(logging_level, base_log_dir)
+    if os.path.exists(config("logging").get("logging_config")):
+        print("Found logging config")
```

Comment:
remove this? we can add a debug logging at the end of `configure_logging` function, to print where the logging configs are coming from

Reply:
Removed and added debug logging.